github/repos_pages.go

```diff
@@ -36,6 +44,45 @@ type PagesBuild struct {
 	UpdatedAt *Timestamp  `json:"updated_at,omitempty"`
 }
 
+// EnablePageSite enables GitHub Pages for the named repo.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/pages/#enable-a-pages-site
+func (s *RepositoriesService) EnablePageSite(ctx context.Context, owner, repo string) (*Pages, *Response, error) {
```

Comment:
I would have gone with `EnablePages` here or `EnablePagesSite` (This goes back to comment regarding consistent naming here).

The produce / service is "GitHub Page***s***", so would include the "s" in "Pages"